bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
@ajw725 What are bin/yarn and bin/update? are those related to the rails 5 upgrade?

Reply:
yes. rails has a rake task you can run to update the version, and it adds whatever new default files ship with rails 5. bin/update is the same as bin/setup, but with a different name. the yarn thing, i'm not exactly sure, but i know rails 5 allows you to manage assets with webpack/yarn if you prefer that to the regular asset pipeline.